home/tests/test_api.py

```diff
@@ -41,12 +50,53 @@ def admin_client(client, django_user_model):
 
 @pytest.fixture()
 def uclient(client, django_user_model):
+    """
+    Access the user interface
+    """
     creds = {"username": "test", "password": "test"}
     django_user_model.objects.create_user(**creds)
     client.login(**creds)
     return client
 
 
+def mk_test_img() -> Image:
+    """
+    make test image
+    """
```

Comment:
For functions where the purpose is adequately expressed in the name, having that repeated in a docstring doesn't really add much value. 🙂